src/crypto/index.js

```diff
@@ -181,6 +189,115 @@ Crypto.prototype.init = async function() {
         );
         this._deviceList.saveIfDirty();
     }
+
+    this._checkAndStartKeyBackup();
+};
+
+/**
+ * Check the server for an active key backup and
+ * if one is present and has a valid signature from
+ * one of the user's verified devices, start backing up
+ * to it.
+ */
+Crypto.prototype._checkAndStartKeyBackup = async function() {
+    console.log("Checking key backup status...");
+    let backupInfo;
+    try {
+        backupInfo = await this._baseApis.getKeyBackupVersion();
+    } catch (e) {
+        console.log("Error checking for active key backup", e);
+        if (e.httpStatus / 100 === 4) {
+            // well that's told us. we won't try again.
+            this._checkedForBackup = true;
+        }
+        return;
+    }
+    this._checkedForBackup = true;
+
+    const trustInfo = await this.isKeyBackupTrusted(backupInfo);
+
+    if (trustInfo.usable && !this.backupInfo) {
+        console.log("Found usable key backup: enabling key backups");
+        this._baseApis.enableKeyBackup(backupInfo);
+    } else if (!trustInfo.usable && this.backupInfo) {
+        console.log("No usable key backup: disabling key backup");
+        this._baseApis.disableKeyBackup();
```

Comment:
Why not managing the case `(trustInfo.usable && this.backupInfo`) ?

If I understand the logic right, it would be:
```
else if (trustInfo.usable && this.backupInfo) {
        console.log("Found usable key backup: use the new key backup version");
        this._baseApis.disableKeyBackup();
        this._baseApis.enableKeyBackup(backupInfo);
    }
```

Reply:
Of course, we should do nothing if the backup version we got from the hs is the same as in `this.backupInfo`.

Reply:
ah sorry - replied to this on matrix but should do so here for completeness: we probably ought to but there's nothing that would actually trigger us to look for this case since we don't get told about new backup version in the sync stream.